package.json

```diff
@@ -41,7 +42,8 @@
     "flow-bin": "^0.55.0",
     "flow-copy-source": "^1.2.1",
     "jest": "^21.2.0",
-    "prettier": "^1.7.1",
+    "jest-cli": "^21.2.0",
```

Comment:
I'll remove it. I added it while trying to understand why jest would not find any tests. Long story short if `app/node_modules/clubhouse-lib/.../__tests__`, jest will check for `app/.../__tests` instead of the lib.

Reply:
Ah, odd. I haven't yet seen that issue. If there's something in this project that's causing it, I'd love to address it.